packages/core/src/alert/alert.element.scss

```diff
@@ -19,6 +19,10 @@ $lightweight-alert-line-height: $cds-global-typography-body-line-height;
   min-height: var(--min-height);
 }
 
+::slotted([cds-text~='link']) {
+  color: var(--color) !important;
```

Comment:
I don't think we want links to be the same color as the text in the lightweight alert style...

Reply:
fixed